web/app/components/new/doc-form.ts

```diff
@@ -13,21 +13,13 @@ import FlashService from "ember-cli-flash/services/flash-messages";
 import { assert } from "@ember/debug";
 import cleanString from "hermes/utils/clean-string";
 import { ProductArea } from "hermes/services/product-areas";
+import { next } from "@ember/runloop";
 
 interface DocFormErrors {
   title: string | null;
-  summary: string | null;
   productAbbreviation: string | null;
-  contributors: string | null;
 }
 
-const FORM_ERRORS: DocFormErrors = {
-  title: null,
-  summary: null,
-  productAbbreviation: null,
-  contributors: null,
-};
```

Comment:
This error logic was broken, but it didn't matter because the disabled state of the button prevented it from being used.